### PR TITLE
Update G-Cloud 9 coming message

### PR DESCRIPTION
**Pull request by TheDoubleK**
For this story: https://www.pivotaltracker.com/story/show/140037777

`&#x2011;` is a non-breaking hyphen: http://www.fileformat.info/info/unicode/char/2011/index.htm

Looks like this:
screen shot 2017-02-16 at 14 37 08

### CODE REVIEW

File: frameworks/g-cloud-9/messages/homepage-sidebar.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 coming:
-  heading: 'G-Cloud&nbsp;9 coming soon'
+  heading: 'G&#x2011;Cloud&nbsp;9 is coming soon'
   messages:
     - 'Provide cloud hosting, software and support.'
-    - 'You need an account to receive notifications about when you can apply.'
+    - 'You need a supplier account so we can email you as soon as G&#x2011;Cloud opens for applications.'
```
Comment (idavidmcdonald) [inline frameworks/g-cloud-9/messages/homepage-sidebar.yml]: Should this say 'as soon as G-Cloud **9** opens'...

Comment (TheDoubleK) [inline frameworks/g-cloud-9/messages/homepage-sidebar.yml]: Not according to the ticket with new words.

Comment (TheDoubleK) [inline frameworks/g-cloud-9/messages/homepage-sidebar.yml]: I did wonder.

Maybe worth asking the question.

**Comment by idavidmcdonald** [comment]
👍

**Pull request merged**